### DISCUSSION TITLE
Is it working on mobile ?

### DISCUSSION THREAD

**Discussion by Tufis16** [Estradel/URP-Simple-Per-Object-Motion-Blur#2] [General]
I'm asking for be sure. Does this work on mobile? I am building and testing it as a very simple car driving game, but it doesn't seem to work.

**Comment by Estradel** [comment]
Hi 👋
Unfornatly, I haven't tested on mobile but only on my Windows desktop. But does this work with a desktop build for you ? I know that web / mobile platforms are not feature complete and maybe they do not support (currently) the new Full Screen feature ? It wouldn't surprise me but I haven't found anything on the Internet to confirm this.

Let me know if you have something new.

P.S. In my tests, I used the forward rendering path. Maybe this helps

**Reply by Tufis16** [reply]
Thank you so much for reply, if i found something i will tell you.
